oracle-linux-image-tools/distr/ol7-slim/provision.sh

```diff
@@ -232,21 +231,26 @@ distr::cleanup() {
   find /boot -name "*.orabackup*" -exec rm -rf {} \;
 
   echo_message "Remove leftover firewall rules"
-  sed -i -e '/-p 50/d' /etc/sysconfig/iptables
-  sed -i -e '/-p 51/d' /etc/sysconfig/iptables
-  sed -i -e '/--dport 5353/d' /etc/sysconfig/iptables
-  sed -i -e '/--dport 631/d' /etc/sysconfig/iptables
-  sed -i -e '/-p 50/d' /etc/sysconfig/ip6tables
-  sed -i -e '/-p 51/d' /etc/sysconfig/ip6tables
-  sed -i -e '/--dport 5353/d' /etc/sysconfig/ip6tables
-  sed -i -e '/--dport 631/d' /etc/sysconfig/ip6tables
+  if [[ -f /etc/sysconfig/iptables ]]; then
+    sed -i -e '/-p 50/d' /etc/sysconfig/iptables
+    sed -i -e '/-p 51/d' /etc/sysconfig/iptables
+    sed -i -e '/--dport 5353/d' /etc/sysconfig/iptables
+    sed -i -e '/--dport 631/d' /etc/sysconfig/iptables
+  fi
+  if [[ -f /etc/sysconfig/ip6tables ]]; then
+    sed -i -e '/-p 50/d' /etc/sysconfig/ip6tables
+    sed -i -e '/-p 51/d' /etc/sysconfig/ip6tables
+    sed -i -e '/--dport 5353/d' /etc/sysconfig/ip6tables
+    sed -i -e '/--dport 631/d' /etc/sysconfig/ip6tables
+  fi
 
   echo_message "Remove MAC addresses"
   for f in /etc/sysconfig/network-scripts/ifcfg-eth*; do
      [ -e "$f" ] && sed -i '/^HWADDR=/d' "$f"
   done
 
   echo_message "Yum cleanup"
+  > /etc/yum/vars/ociregion
   yum clean all --enablerepo=\*
```

Comment:
This line is redundant considering the next line.

Reply:
Am I missing something?
```
[root@bommel ~]# cat /etc/yum/vars/ociregion
-frankfurt
[root@bommel ~]# yum clean all --enablerepo=\*
Loaded plugins: langpacks, ulninfo
Cleaning repos: ksplice-uptrack ol7_MODRHCK ol7_UEKR3 ol7_UEKR3_OFED20 ol7_UEKR4 ol7_UEKR4_OFED ol7_UEKR4_archive ol7_UEKR5 ol7_UEKR5_RDMA
              : ol7_UEKR5_archive ol7_addons ol7_developer ol7_developer_EPEL ol7_developer_UEKR5 ol7_developer_kvm_utils ol7_ksplice
              : ol7_kvm_utils ol7_latest ol7_latest_archive ol7_oci_included ol7_ociyum_config ol7_optional_archive ol7_optional_latest
              : ol7_oracle_instantclient ol7_security_validation ol7_software_collections ol7_u0_base ol7_u1_base ol7_u2_base ol7_u3_base
              : ol7_u4_base ol7_u5_base ol7_u6_base ol7_u7_base ol7_x86_64_userspace_ksplice
[root@bommel ~]# cat /etc/yum/vars/ociregion
-frankfurt
```

Reply:
You're not missing anything. I put my comment on the wrong line! I mean the `yum clean all` line is redundant considering you're then running `rm -rf /var/cache/yum/*`

Reply:
Right -- 067b58f